docs/src/views/ColorPalette.tsx

```diff
@@ -47,78 +96,139 @@ const ColorPalette = () => (
+            common needs. This decision is experienced from Material UI and allows to define more
+            variants than semantical naming (lightest, lighter, etc.). The reason why we don't
+            require all colors to be implemented, is just becase some application may not use some
+            of these colors, so we don't want to add overheader for defining things which are not
+            necessary.
           </p>
 
           <Grid columns={2} variables={{ gridGap: '2rem' }}>
             {_.map(naturalColors, (variants, color) => (
               <div key={color}>
-                <ColorBox name={color} rounded value={colors[color][500]} />
+                <ColorBox name={color} rounded value={colors[color][600]} />
               </div>
             ))}
           </Grid>
 
-          <Header as="h2">Emphasis colors</Header>
-          <p>This part of the palette includes primary and secondary colors.</p>
+          <Header as="h3">Emphasis colors</Header>
+          <p>This part of the palette may includes primary and secondary colors.</p>
 
           <Grid columns={2}>
             {_.map(emphasisColors, (variants, color) => (
               <div key={color}>
-                <ColorBox name={color} rounded size="big" value={colors[color][500]} />
+                <ColorBox name={color} rounded size="big" value={colors[color][600]} />
               </div>
             ))}
           </Grid>
 
-          <Header as="h2">Contextual colors</Header>
-          <p>
-            Contextual colors can be used to provide "meaning through colors", however they can be
-            just aliases for natural colors.
-          </p>
-
-          <Grid columns={2}>
-            {_.map(contextualColors, (variants, color) => (
+          <Header as="h3">All colors</Header>
+          <p>This are all colors available in the color palette.</p>
+          <Grid columns={2} variables={{ gridGap: '2rem' }}>
+            {_.map({ ...emphasisColors, ...naturalColors }, (variants, color) => (
               <div key={color}>
-                <ColorBox name={color} rounded size="big" value={colors[color][500]} />
+                <ColorVariants name={color} />
               </div>
             ))}
           </Grid>
 
-          <Header as="h2">Text colors</Header>
+          <Header as="h2">Color scheme</Header>
+          <p>
+            Now that we know how the color palette is defined, let's see how we can use these
+            values, so that they will work correctly when different themes will be applied. As part
+            of each theme, we are defining <b>color scheme</b>, which will define the design tokens
+            usages of the different colors from the palette, that make sense for the developers. For
+            example, let's take a look of one color scheme defined for primary in light theme and
+            high contrast theme.
```

Comment:
I want to mention that Stardust contains Teams stuff, but it's not required:

```suggestion
            Teams light as contrast themes.
```

Reply:
- let's take a look of one color scheme defined for primary in light theme and high contrast theme.
+let's take a look of one color scheme defined for primary in Teams light theme and high contrast themes.

Did you mean this?